docs/ota-client-guide/modules/ROOT/examples/sota-local.toml

```diff
@@ -0,0 +1,15 @@
+[provision]
+provision_path = "credentials.zip"
+primary_ecu_hardware_id = "local-fake"
+
+[uptane]
+secondary_configs_dir = "secondary_configs"
+
+[logger]
+loglevel = 1
+
+[storage]
+path = "storage"
+
+[pacman]
+type = "none"
```

Comment:
So do we really have to duplicate this file from `https://github.com/advancedtelematic/aktualizr/blob/master/config/sota-local.toml`? It would be really nice if we didn't have to maintain both. The one I've linked to is not actually used anywhere as far as I can tell, but as a developer, it is a convenient reference to have in the same directory as everything else. A softlink would be an option, I think.

But more importantly, this version of the config is out of date. `secondary_configs_dir` has been replaced.

Reply:
> So do we really have to duplicate this file 
For the time being, until https://saeljira.it.here.com/browse/OTA-3569 is done.

> secondary_configs_dir has been replaced.
OK, I'll fix in a followup PR